docs/xdr/features/detect/rules_catalog.md

```diff
@@ -53,16 +48,25 @@ Description of each effort level:
 - `Advanced`: rule could require more effort to be enabled and could raise alerts frequently depending on the IT configuration
 - `Master`: rule could require a specific configuration to be enabled and/or could raise a high number of alerts, but is designed to detect weaker signals. `Master` rules usually require an additional customization effort, depending on the IT context and configuration. They are designed for more mature organizations. 
 
-#### Capabilities
+#### Intake formats
+
+Rules are associated with Intake formats that they are compatible with. Detection rules can be enabled when they have a compatible intake format configured that provide the necessary data to detect suspicious activites.
```

Comment:
Rules are linked to specific intake formats with which they are compatible. Detection rules can be enabled when a compatible intake format is configured, providing the essential data needed to identify suspicious activities.